Makefile.dev

```diff
@@ -242,6 +242,14 @@ build-all: build-app create-build-config build-container set-app-image build-ima
 deploy-changes:
 	./deploy-changes.sh
 
+setup-deps:
```

Comment:
awkward use of Makefile (non-portable)

Reply:
not sure I got your comment.... what is yours suggestion?

Reply:
I don't think installing os-level dependencies belongs in a Makefile

Reply:
how about a bash script outside the makefile? the makefile will trigger it

Reply:
i think it's fine. it's an example, it does the job